### ISSUE TITLE
open_basedir after php update

### ISSUE THREAD

**Issue by Tealk** [BookStackApp/BookStack#2836]
Since the last php update yesterday or the day before, bookstack no longer works.
I have had to add the following two paths so that bookstack works again
`open_basedir = ".../de:/de_informal"`

But the two folders /de and /de_informal do not exist in the root directory, how come they are referenced?

error.log:
```
2021/07/03 09:56:02 [error] 29923#29923: *2611339 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught ErrorException: file_exists(): open_basedir restriction in effect. File(/de/errors.php) is not within the allowed path(s): (/var/customers/webs/Anzah/books.rollenspiel.monster:/var/customers/tmp/Anzah:/usr/share/php:/usr/share/php5:/tmp) in /var/customers/webs/Anzah/books.rollenspiel.monster/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php:23
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError()
#1 /var/customers/webs/Anzah/books.rollenspiel.monster/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(23): file_exists()
#2 /var/customers/webs/Anzah/books.rollenspiel.monster/vendor/laravel/framework/src/Illuminate/Translation/FileLoader.php(122): Illuminate\Filesystem\Filesystem->exists()
#3 /var/customers/webs/Anzah/books.rollenspiel.monster/app/Translation/FileLoader.php(23): Illuminate\Translation\FileLoader->loadPath()
#4 /var/customers/webs/Anzah/books.rollenspiel.monster/vendor...PHP message: PHP Fatal error:  Uncaught ErrorException: file_exists(): open_basedir restriction in effect. File(/de/errors.php) is not within the allowed path(s): (/var/customers/webs/Anzah/books.rollenspiel.monster:/var/customers/tmp/Anzah:/usr/share/php:/usr/share/php5:/tmp) in /var/customers/webs/Anzah/books.rollenspiel.monster/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php:23
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError()
#1 /var/customers/webs/Anzah/books.rollenspiel.monster/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(23): file_exists()
#2 /var/customers/webs/Anzah/books.rollenspiel.monster/vendor/laravel/framework/src/Illuminate/Translation/FileLoader.php(122): Illuminate\Filesystem\Filesystem->exists()
#3 /var/customers/webs/Anzah/books.rollenspiel.monster/app/Translation/FileLoader.php(23): Illuminate\Translation\FileLoader->loadPath()
#4 /va
2021/07/03 09:56:16 [error] 29923#29923: *2611376 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught ErrorException: file_exists(): open_basedir restriction in effect. File(/de_informal/errors.php) is not within the allowed path(s): (/var/customers/webs/Anzah/books.rollenspiel.monster:/var/customers/tmp/Anzah:/usr/share/php:/usr/share/php5:/tmp:/de) in /var/customers/webs/Anzah/books.rollenspiel.monster/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php:23
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError()
#1 /var/customers/webs/Anzah/books.rollenspiel.monster/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(23): file_exists()
#2 /var/customers/webs/Anzah/books.rollenspiel.monster/vendor/laravel/framework/src/Illuminate/Translation/FileLoader.php(122): Illuminate\Filesystem\Filesystem->exists()
#3 /var/customers/webs/Anzah/books.rollenspiel.monster/app/Translation/FileLoader.php(23): Illuminate\Translation\FileLoader->loadPath()
#4 /var/customers/webs/Anzah/books.rollenspiel.m...PHP message: PHP Fatal error:  Uncaught ErrorException: file_exists(): open_basedir restriction in effect. File(/de_informal/errors.php) is not within the allowed path(s): (/var/customers/webs/Anzah/books.rollenspiel.monster:/var/customers/tmp/Anzah:/usr/share/php:/usr/share/php5:/tmp:/de) in /var/customers/webs/Anzah/books.rollenspiel.monster/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php:23
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError()
#1 /var/customers/webs/Anzah/books.rollenspiel.monster/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(23): file_exists()
#2 /var/customers/webs/Anzah/books.rollenspiel.monster/vendor/laravel/framework/src/Illuminate/Translation/FileLoader.php(122): Illuminate\Filesystem\Filesystem->exists()
#3 /var/customers/webs/Anzah/books.rollenspiel.monster/app/Translation/FileLoader.php(23): Illuminate\Translation\FileLoader->load
```

**Comment by ssddanbrown** [comment]
Thanks for raising @Tealk. This helped highlight an error in how we loaded translation files when a theme was not in use.
I've patched this within 4da72aa267903dcb448fe15dd5637391a3f440e8 and this is part of [BookStack v21.05.3](https://github.com/BookStackApp/BookStack/releases/tag/v21.05.3) which I've just published. 

I'll therefore close this off but please report back if that release does not seem to fix the issue for you.

**Comment by Tealk** [comment]
error is fixed, thanks for the fast work
